.github/ISSUE_TEMPLATE/DISCUSSION.md

```diff
@@ -0,0 +1,10 @@
+---
+name: discussion issue template
+about: template for discussion
+title: ' '
+labels: ' '
+assignees: ' '
+
+---
+####Discussion Topic:[your topic here]
```

Comment:
What is the "####" notation for? Is that a markdown sub-sub-sub-subsection?

Reply:
In short, yes. I'm just using it because it makes a smaller heading that takes up less space than a conventional single pound sign.

Reply:
Hmm... Ok. I think for that to work you might need a space between the # and the title.

However, if you want to emphasize the "heading" and not make it too big, I'd suggest just making it bold using ** characters: `**Discussion Topic**`

Reply:
Changed all the headers to bold in the latest update.